config/src/config/secure_backend_config.rs

```diff
@@ -41,6 +41,10 @@ pub struct VaultConfig {
     /// a secret, S, without a namespace would be available in secret/data/S, with a namespace, N, it
     /// would be in secret/data/N/S.
     pub namespace: Option<String>,
+    /// Vault leverages leases on many tokens, specify this to automatically have your lease
+    /// renewed up to that many seconds more. If this is not specified, the lease will not
+    /// automatically be renewed.
+    pub renew_ttl: Option<u32>,
```

Comment:
nit: any chance we could rename this something like "renew_ttl_secs"? Just to be explicit that it is in seconds? It seems like this comment is the only thing that makes this obvious?